src/common/qprojectm_mainwindow.cpp

```diff
@@ -232,7 +232,7 @@ projectm* QProjectM_MainWindow::GetProjectM()
 
 void QProjectM_MainWindow::addPCM(float * buffer, unsigned int bufferSize)
 {
-    projectm_pcm_add_float_2ch_data(qprojectM()->instance(), buffer, bufferSize);
+    projectm_pcm_add_float(qprojectM()->instance(), buffer, bufferSize / 2, PROJECTM_STEREO);
```

Comment:
That's probably not really working, as there's no way to get the real sample format or channel count at this point. The whole audio capturing code requires a rework, so I'll leave it in this state as it doesn't crash at the very least.